site/content/docs/5.0/components/alerts.md

```diff
@@ -206,15 +206,15 @@ This makes an alert listen for click events on descendant elements which have th
         <code>getInstance</code>
       </td>
       <td>
-        Static method which allows you to get the alert instance associated to a DOM element, you can use it like this: <code>bootstrap.Alert.getInstance(alert)</code>
+        Static method which allows you to get the alert instance associated to a CSS selector, DOM element, or jQuery object, you can use it like this: <code>bootstrap.Alert.getInstance(alert)</code>
```

Comment:
Ah, the instance is always associated with the DOM element only, not with the CSS selector or jquery object.

Reply:
My apologies, but I am unsure what I should be revising here. Are you referring to the documentation language, the code example, or something else?

Reply:
Knowing from inside info that @rohit2sharma95  is planning to re-organize documentation, I would suggest to keep only the js and test things, reverting doc changes 

PS: Rohit means: 
`getInstance` gets as argument a `a CSS selector, DOM element, or jQuery object,` and returns the instance associated to the DOM element

Reply:
True @GeoSot 👍